packages/backend/src/types/express/index.d.ts

```diff
@@ -0,0 +1,12 @@
+export {}
+
+declare global {
+  namespace Express {
+    interface User {
+      id: string
```

Comment:
I belive this is extending the Request express type to also contain the user,right?
If such, add email aswell, since it is also on the req.user now